source/jormungandr/jormungandr/default_settings.py

```diff
@@ -77,6 +77,16 @@
     }
 }
 
+AUTOCOMPLETE = {
```

Comment:
maybe the default settings should be None for the moment no ? (but you can copy/paste this configuration example in the PR description)